modules/develop/pages/produce-data/leader-pinning.adoc

```diff
@@ -0,0 +1,43 @@
+= Leader Pinning
+:page-categories: Clients, Development
+// tag::single-source[]
+:description: Learn about leader pinning and how to configure a preferred partition leader based on availability zones.
+
+Produce requests to Redpanda topics go through the topic partition leader, which then ensure that messages are synced across its follower replicas. For a Redpanda cluster deployed across multiple availability zones (AZs), leader pinning is a mechanism to ensure that a topic's partition leaders are geographically closer to clients. Leader pinning can lower networking costs and help guarantee lower latency by routing produce requests and consume requests without xref:develop:consume-data/follower-fetching.adoc[follower fetching] to brokers located in certain AZs.
```

Comment:
@ztlpn Is the rephrase accurate?

Reply:
Well, if by "consume requests without follower fetching" it means "consume requests without `client.rack` property", then it is correct, but it feels like it is juxtaposing leader pinning and follower fetching, even though they are not exclusive and can be used together.

Maybe we can just say "produce and consume requests", and then mention follower fetching in the following sentence as a feature that can also reduce networking costs, but only for consume requests.